client/consensus/beefy/src/tests.rs

```diff
@@ -161,23 +161,26 @@ impl BeefyTestNet {
 		// push genesis to make indexing human readable (index equals to block number)
 		all_hashes.push(self.peer(0).client().info().genesis_hash);
 
-		let built_hashes = self.peer(0).generate_blocks(count, BlockOrigin::File, |builder| {
-			let mut block = builder.build().unwrap().block;
-
-			if include_mmr_digest {
-				let block_num = *block.header.number();
-				let num_byte = block_num.to_le_bytes().into_iter().next().unwrap();
-				let mmr_root = MmrRootHash::repeat_byte(num_byte);
-				add_mmr_digest(&mut block.header, mmr_root);
-			}
-
-			if *block.header.number() % session_length == 0 {
-				add_auth_change_digest(&mut block.header, validator_set.clone());
-			}
-
-			block
-		});
-		all_hashes.extend(built_hashes);
+		for block_num in 0..count {
+			let block_num: NumberFor<Block> = block_num.saturating_add(1).try_into().unwrap();
```

Comment:
would be nice for `block_num` here to be tied to actual block number; so that if this function is called multiple times, it correctly adds new blocks with correct "mmr number" on top - but it's just a nice to have, not requirement

Reply:
done: https://github.com/paritytech/substrate/pull/13737/commits/6443a9dd8f7fc5e0cf753211b3f050b6ddd33421